### PR TITLE
Fix pagination for video endpoint

### PR DESCRIPTION
**Pull request by LionOps**

### CODE REVIEW

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

File: src/response-handler.js [hunk 1]
```diff
@@ -18,7 +18,7 @@ class ResponseHandler {
   success() {
     const data = JSON.parse(this.res.xhr.responseText);
     data.page = this.page;
-    data.totalPages = Math.ceil(data.totalHits / this.perPage);
+    data.totalPages = Math.ceil(data.totalHits || data.total / this.perPage);
```
Comment (beck) [inline src/response-handler.js]: Won't this always resolve to `data.totalHits`?

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

File: src/response-handler.js (hunk 1 above)
Comment (LionOps) [reply src/response-handler.js]: Yeah. Fixed.

**Review by beck — APPROVED** [review]
For my own edification, what's the diff between `.totalHits` and `.total`?

**Comment by LionOps** [comment]
> For my own edification, what's the diff between .totalHits and .total?

`total` is the absolute total  whereas `totalHits` is the total the API is willing to let you query. For images, `totalHits` normally caps at 500. Videos have no cap currently.

**Pull request merged**